src/openrct2/ride/Ride.cpp

```diff
@@ -4347,7 +4347,7 @@ int32_t RideGetRandomColourPresetIndex(ride_type_t rideType)
  *
  *  Based on rct2: 0x006B4776
  */
-void Ride::setColourPreset(uint8_t index)
+void Ride::setColourPreset(uint8_t index, uint8_t indexVeh)
```

Comment:
`indexVeh` is not remotely making clear what it does. The code at line +4360-4364 isn’t either.